src/main/webapp/script.js

```diff
@@ -260,3 +280,137 @@ function checkPercentage(headerColumn) {
   }
   return false;
 }
+
+// Returns an object containing all of the relevant data in order
+// to render the geoJson map of the counties.
+function getMapsData(censusDataArray) {
+  const countyToPopMap = new Map();
+  const populationsList = [];
+  // Get rid of the header
+  const censusArray = censusDataArray.slice(1);
+  Array.prototype.forEach.call(censusArray, (county) => {
+    // The current county strings are in a layout like this:
+    // "Contra Costa County, California"
+    // and we need to get them like this "Contra Costa"
+    const countyAndStateArray = county[0].split(',');
```

Comment:
depending on a specific format of text is risky. We can submit this for now but discuss how we can me this more robust.

Reply:
Kyra pointed out that there is data in the amCharts geoJson that I can use.